public/app/plugins/panel/stat/module.tsx

```diff
@@ -1,11 +1,26 @@
-import { BigValueTextMode, commonOptionsBuilder, sharedSingleStatMigrationHandler } from '@grafana/ui';
-import { PanelPlugin } from '@grafana/data';
+import {
+  BigValueColorMode,
+  BigValueTextMode,
+  commonOptionsBuilder,
+  sharedSingleStatMigrationHandler,
+} from '@grafana/ui';
+import { FieldConfigProperty, PanelPlugin } from '@grafana/data';
 import { addOrientationOption, addStandardDataReduceOptions, StatPanelOptions } from './types';
 import { StatPanel } from './StatPanel';
 import { statPanelChangedHandler } from './StatMigrations';
 
 export const plugin = new PanelPlugin<StatPanelOptions>(StatPanel)
-  .useFieldConfig()
+  .useFieldConfig({
+    standardOptions: {
+      [FieldConfigProperty.Color]: {
+        settings: {
+          byValueSupport: true,
+          bySeriesSupport: true,
+          preferThresholdsMode: false,
```

Comment:
Why this change? Think it will make it harder to switch to this panel from the graphs. You will always have to change color scheme from palette every time

Reply:
yes, I tested this and this does indeed make stat panels use palette by default (so new stat panels are not yellow)

Reply:
:(   any ideas how to have the fixed color stick when we add it in migrations?

https://github.com/grafana/grafana/pull/35538/files#diff-8d7549660e15a6f88a67d7c95cae4e98c7602cae18279e227f93e5ffa37841b7R32

I looked everywhere and finally found that without the settings above it would remove the things I just set.

Reply:
Ah missed be bad logic in the change plugin option handling, if the color is set to fixed color the that should not be changed